ts/image-occlusion/mask-editor.ts

```diff
@@ -129,10 +118,49 @@ function initCanvas(onChange: () => void): fabric.Canvas {
         onChange();
     });
     canvas.on("object:removed", onChange);
-    setCenterXForZoom(canvas);
     return canvas;
 }
 
+const setupBoundingBox = (canvas: fabric.Canvas, size: Size, image: HTMLImageElement): Promise<fabric.Rect> => {
+    return new Promise((resolve) => {
+        const boundingBox = new fabric.Rect({
+            fill: "transparent",
+            width: size.width,
+            height: size.height,
+            hasBorders: false,
+            hasControls: false,
+            lockMovementX: true,
+            lockMovementY: true,
+            evented: false,
+            stroke: "red",
+        });
+        globalThis.boundingBox = boundingBox;
```

Comment:
We should avoid polluting the global namespace where possible, especially with generic names like this. You've already added a getBoundingBox() function - could we make it use a file-private variable instead of a global?

Reply:
I will remove global namespace, got busy but in next commit the issues will be resolved.